external/fv3fit/tests/test_sklearn_wrapper.py

```diff
@@ -164,6 +164,35 @@ def test_predict_returns_unstacked_dims():
     assert prediction.dims == input_data.dims
 
 
+def fit_wrapper_with_gridcell_data():
+    model = LinearRegression()
+    wrapper = SklearnWrapper(
+        input_variables=["a"],
+        output_variables=["b"],
+        model=model,
+        predict_columns=False,
+    )
+    dims = [
+        "sample",
+    ]
+    shape = (10,)
+    arr = np.arange(shape[0])
+    input_data = xr.Dataset({"a": (dims, arr), "b": (dims, arr + 1.0)})
+    wrapper.fit(tfdataset_from_batches([input_data]))
+    return input_data, wrapper
+
+
+def test_predict_columns_false():
+    _, wrapper = fit_wrapper_with_gridcell_data()
+    input_data = get_unstacked_data()
+    # works
+    _ = wrapper.predict(input_data)
+    wrapper.predict_columns = True
+    with pytest.raises(ValueError):
+        # wrong number of features
+        _ = wrapper.predict(input_data)
+
+
```

Comment:
Could you include a test where the trained model output a scalar for each z level point and the wrapper does the correct unstacking so that the predicted dataset has the full column of z predicted?